md/single-sign-on-with-cas.md

```diff
@@ -54,8 +53,8 @@ For a standard installation, it is not necessary to modify this file.
 To configure Bonita BPM Engine for CAS:
 
 1. If you do not already have it, download the Subscription edition deploy zip from the customer portal.
-2. Add the CAS module. To do this, copy cas-3.3.1-module/org to WILDFLY_HOME/modules to merge the CAS module with the existing modules.
-3. Make the CAS module global so that it can be used by any application. To do this, edit `WILDFLY_HOME/standalone/configuration/standalone.xml` and change the definition of the `ee` subsystem to the following:
+2. Add the CAS module. To do this, copy `BonitaBPMSubscription-7.4.x-deploy/cas-3.3.1-module/org` to `WILDFLY_HOME/server/modules` to merge the CAS module with the existing modules.
```

Comment:
would be better to put BonitaBPMSubscription-x.x.x-deploy/ IMO so that we don't need to change the version on each branch